### ISSUE TITLE
Site down?

### ISSUE THREAD

**Issue by j-f1** [pierce403/cors.io#5]
The website https://cors.io appears to be down.

**Comment by nizarmah** [comment]
Yes, I believe it is because pierece made it possible for you to host your own proxy. Just follow the steps on the README and you should be ready to go.

@pierce403 do you plan to bring the website up again ?

**Comment by marcosscriven** [comment]
Just came here to check the same. Looks like it's been a while now @pierce403 - https://twitter.com/nektro/status/1143950416062926848

Totally don't expect you to support this - but can you confirm there's no plans to keep this going?
